catalog/spatial/registry/registry-publication-action-provider/src/main/java/org/codice/ddf/registry/publication/manager/RegistryPublicationManager.java

```diff
@@ -70,7 +70,10 @@ public void handleEvent(Event event) {
   }
 
   public void init() {
```

Comment:
❓ Does waiting 1 millisecond to call setPublications() really help?

Reply:
I was under the understanding that you had to provide a delay with this `ScheduledExecutorService`

Reply:
A `ScheduledExecutorService` is an `ExecutorService`, so it does have the `invoke()` and `submit()` method that doesn't wait.

Reply:
I can change it to use `submit()` instead for the single runnable